frontend/viewer/src/ProjectView.svelte

```diff
@@ -311,6 +320,22 @@
 
                   </div>
                 {/if}
+                {#if $features.openWithFlex && $selectedEntry}
+                  <div class="contents">
+<!--                    button must be a link otherwise it won't follow the redirect to a protocol handler-->
+                    <Button
```

Comment:
It's slightly confusing that we have more than one Button component in different parts of our frontend; it took me a minute to realize that this is the  one from svelte-ux rather than the Button.svelte from our `frontend/src/lib/forms`, which doesn't have an `href` prop to auto-switch from `<button>` to `<a>`. Though since the viewer has a completely different look than the LexBox UI, it's not a bad thing that there are two different UX frameworks in use. Just mildly confusing.

(This comment is just informational; no action needed before resolving).